src/sentry/tasks/post_process.py

```diff
@@ -336,53 +364,52 @@ def post_process_group(
 
             return
 
+        group_states = kwargs.get("group_states")
+
+        if group_states is None:
+            # error issue
+            group_states = {
+                1: {
+                    "id": group_id,
+                    "is_new": is_new,
+                    "is_regression": is_regression,
+                    "is_new_group_environment": is_new_group_environment,
+                }
+            }
+        else:
+            # performance issue
+            return
```

Comment:
It might be safer here to always include the `group_id` etc from the existing params in group states. We should only pass `group_states` for perf issues, but I just want to guard against anything going wrong. I could imagine that in the future, maybe we pass `group_states` as an empty list accidentally, and then we end up not including the error issue here.

Reply:
looking at the forwarding function it looked like that data was always sent in the `group_states` param, so I didn't want to double process anything by blind adding that info to the dict